src/quo/components/profile/collectible_list_item/view.cljs

```diff
@@ -10,13 +10,54 @@
     [quo.foundations.gradients :as gradients]
     [quo.theme]
     [react-native.core :as rn]
+    [react-native.reanimated :as reanimated]
     [schema.core :as schema]
+    [utils.datetime :as datetime]
     [utils.i18n :as i18n]))
 
+(def timing-options-out 650)
+(def timing-options-in 1000)
+(def first-load-time 500)
+(def cached-load-time 200)
```

Comment:
curious about how this time was estimated

Reply:
Just visual experimentation really

Reply:
Okay, would it make sense if this was moved to a constant file?